cmd/fetch.go

```diff
@@ -476,3 +504,54 @@ func fetchAvailCapacity(cmd *cobra.Command, args []string) {
 		fmt.Println(capacity)
 	}
 }
+
+//fetchTasksWithStatus returns lists of tasks for a given set of status
+func fetchTasksWithStatus(cmd *cobra.Command, args []string) {
+	status := *taskStatus
+
+	log.Infof("Fetching tasks for environment/role/job:[%s/%s/%s] \n", *env, *role, *name)
+	log.Infof("Fetching tasks for a given status: %v \n", status)
+
+	// This Query takes nil for values it shouldn't need to match against.
+	// This allows us to potentially avoid expensive calls for specific environments, roles, or job names.
+	if *env == "" {
+		env = nil
+	}
+	if *role == "" {
+		role = nil
+	}
+	if *name == "" {
+		name = nil
+	}
+	queryStatuses, err := scheduleStatusFromString(status)
+	if err != nil {
+		log.Fatalf("error: %+v", err)
+	}
+
+	taskQuery := &aurora.TaskQuery{Environment: env, Role: role, JobName: name, Statuses: queryStatuses}
+
+	tasks, err := client.GetTasksWithoutConfigs(taskQuery)
+	if err != nil {
+		log.Fatalf("error: %+v", err)
+	}
+
+	if toJson {
+		tasksMap := map[string][]*aurora.ScheduledTask{strings.ToUpper(status): tasks}
+		fmt.Println(internal.ToJSON(tasksMap))
+	} else {
+		fmt.Printf("Tasks for status %s:\n", strings.ToUpper(status))
+		for _, t := range tasks {
+			fmt.Println(t)
+		}
+	}
+}
+
+// Convert status slice into ScheduleStatus slice
+func scheduleStatusFromString(status string) ([]aurora.ScheduleStatus, error) {
```

Comment:
I don't think we need this separate function for 3 lines of code and its not used anywhere else.